src/with_prefix.rs

```diff
@@ -8,6 +8,8 @@ use std::fmt;
 /// Serialize with an added prefix on every field name and deserialize by
 /// trimming away the prefix.
 ///
+/// You can set the visibility of the generated module by prefix the module name with a module visibility.
```

Comment:
Maybe you could show an example like `with_prefix!(pub(crate) prefix_foo "foo_");` and mention that it is optional.

Reply:
I've used one of the existing prefixes in the example to do so.
Maybe you can be more specific if I have to create a full example like the Match - Player one below or just showcase a non-runnable example here?

Reply:
Just show a small non-runnable example of the syntax here. Similar to the sentence in the example. Such that you don't have to read the example to know that this is a possibility.